From 78fb7e37eb8bb08ae537d6c487996ff17c810332 Mon Sep 17 00:00:00 2001
From: Trevor Saunders <tbsaunde+binutils@tbsaunde.org>
Date: Mon, 26 Sep 2016 12:42:11 -0400
Subject: [PATCH] tc-xtensa.c: fixup xg_reverse_shift_count typo

gas/ChangeLog:

2016-09-26  Trevor Saunders  <tbsaunde+binutils@tbsaunde.org>

	* config/tc-xtensa.c (xg_reverse_shift_count): Pass cnt_arg instead of
	cnt_argp to concat.
---
 gas/ChangeLog          | 5 +++++
 gas/config/tc-xtensa.c | 2 +-
 2 files changed, 6 insertions(+), 1 deletion(-)

[[ ChangeLog skipped, fails to apply on 2.27 ]]
diff --git a/gas/config/tc-xtensa.c b/gas/config/tc-xtensa.c
index d062044..ca261ae 100644
--- a/gas/config/tc-xtensa.c
+++ b/gas/config/tc-xtensa.c
@@ -2228,7 +2228,7 @@ xg_reverse_shift_count (char **cnt_argp)
   cnt_arg = *cnt_argp;
 
   /* replace the argument with "31-(argument)" */
-  new_arg = concat ("31-(", cnt_argp, ")", (char *) NULL);
+  new_arg = concat ("31-(", cnt_arg, ")", (char *) NULL);
 
   free (cnt_arg);
   *cnt_argp = new_arg;
-- 
2.9.3

